### PR TITLE
fix: Remove support for MacOs

### PR DESCRIPTION
**Pull request by jorelosorio**
Closes: https://github.com/AtB-AS/kundevendt/issues/11303

> NOTE: This PR also upgrades Debug min iOS Version from 12.2 to 12.4 as it was on Release. I didn't find a reason to have it out of synch, if you know one please let me know.

Before:

image

Now:

image

### CODE REVIEW

File: ios/atb.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -53,7 +53,7 @@
 		6DAA1D0A2A77D3EC00CC8016 /* PassKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 6DAA1D092A77D3EC00CC8016 /* PassKit.framework */; };
 		6DB88ADA2A7A35A10060CC31 /* PassPresentationBridge.m in Sources */ = {isa = PBXBuildFile; fileRef = 6DB88AD92A7A35A10060CC31 /* PassPresentationBridge.m */; };
 		6DB88ADC2A7A55570060CC31 /* PassPresentation.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6DB88ADB2A7A55570060CC31 /* PassPresentation.swift */; };
-		86ADA222293F651800F1E080 /* BuildFile in Frameworks */ = {isa = PBXBuildFile; };
```
Comment (hanne-at-atb) [inline ios/atb.xcodeproj/project.pbxproj:56]: Are these changes intentional? BuildFile -> (null) 🌵

**Review by hanne-at-atb — APPROVED** [review]
Well done! 🚫 💻  ✅ ☎️

**Review by marius-at-atb — APPROVED** [review]
:+1:

File: ios/atb.xcodeproj/project.pbxproj (hunk 1 above)
Comment (jorelosorio) [reply ios/atb.xcodeproj/project.pbxproj:56]: That is created by Xcode.
